src/effects/Bars.cpp

```diff
@@ -196,6 +196,9 @@ std::string Bars::PropertiesJSON(int64_t requested_frame) const {
 	root["right"] = add_property_json("Right Size", right.GetValue(requested_frame), "float", "", &right, 0.0, 0.5, false, requested_frame);
 	root["bottom"] = add_property_json("Bottom Size", bottom.GetValue(requested_frame), "float", "", &bottom, 0.0, 0.5, false, requested_frame);
 
+	// Set the parent effect which properties this effect will inherit
+	root["parent_effect_id"] = add_property_json("Parent", 0.0, "string", info.parent_effect_id, NULL, -1, -1, false, requested_frame);
```

Comment:
Could this be incorporated into the Base class of Effects or Clips? Just curious...